packages/dt_for_itables/add_host_to_root.py

```diff
@@ -0,0 +1,91 @@
+#! /usr/bin/env python3
+"""This is a Python script that modifies the dt_bundle.css to
+add :host to each :root selector. This ensures that the styles
+works in Marimo which uses Shadow DOM.
+"""
```

Comment:
@AllanJard this is a script that adds `host:` selectors for every `root:` selector found in a given CSS file. At the current stage of this PR, I run this script on the CSS bundle produced by `dt_for_itables`, in a post process step.

That's fine with me but before merging I wanted to see what you think of the alternative, that would be to add explicit `host:` selectors in all the datatable CSS files. Is that something you would consider? If so I could use this script to prepare a PR (what would be the appropriate repo(s) to do the PR on, given that I use datatables and a few extensions?)